project-base/app/autoload.php

```diff
@@ -4,6 +4,10 @@
 
 use Doctrine\Common\Annotations\AnnotationRegistry;
 
+$symfonyDumpFunctionPath = 'vendor/symfony/var-dumper/Resources/functions/dump.php';
+
+file_exists(__DIR__ . '/../' . $symfonyDumpFunctionPath) ? require_once __DIR__ . '/../' . $symfonyDumpFunctionPath : require_once __DIR__ . '/../../' . $symfonyDumpFunctionPath;
```

Comment:
this might kill whole application if `var-dumper` will not be installed. 
What about two separate file_exists conditions (monorepo/project)?

Reply:
ab1f68532e84203045c75fd909c23c62e0ceb241